app/models/manageiq/providers/vmware/refresh_helper_methods.rb

```diff
@@ -0,0 +1,21 @@
+module ManageIQ::Providers::Vmware::RefreshHelperMethods
+  extend ActiveSupport::Concern
+
+	def process_collection(collection, key)
+    @data[key] ||= []
```

Comment:
Guys, why are you reviewing 3 minutes old commits when there was already a new one ;).

Sorry for that, as said below, I missed it in my editor until I checked files on Github.